src/repl/mod.rs

```diff
@@ -70,16 +70,12 @@ pub struct REPLImpl {
     cache: Cache,
     /// The parser, supporting toplevel let declaration.
     parser: grammar::ExtendedTermParser,
-    /// The eval environment. Contain the global environment with the stdlib, plus toplevel
+    /// The global environment. Contain the global environment with the stdlib, plus toplevel
     /// declarations and loadings made inside the REPL.
-    eval_env: eval::Environment,
-    /// The initial eval environment, without the toplevel declarations made inside the REPL. Used
+    env: GlobalEnv,
+    /// The initial type environment, without the toplevel declarations made inside the REPL. Used
     /// to typecheck imports in a fresh environment.
-    init_eval_env: eval::Environment,
-    /// The typing environment, counterpart of the eval environment for typechecking. Entries are
-    /// [`TypeWrapper`](../typecheck/enum.TypeWrapper.html) for the ease of interacting with the
-    /// typechecker, but there are not any unification variable in it.
```

Comment:
I think you deleted the documentation by error?

Reply:
Actualy it has been moved to `cache::GlobalEnv::type`